app/test/block_size_test.go

```diff
@@ -247,6 +250,28 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForBlob() {
 	}
 }
 
+func (s *IntegrationTestSuite) TestUnwrappedPFBRejection() {
+	t := s.T()
+	val := s.network.Validators[0]
+
+	blobTx := blobfactory.RandBlobTxsWithAccounts(
+		s.cfg.TxConfig.TxEncoder(),
+		s.kr,
+		val.ClientCtx.GRPCClient,
+		100000,
+		false,
+		s.cfg.ChainID,
+		s.accounts[:1],
+	)
+
+	btx, isBlob := coretypes.UnmarshalBlobTx(blobTx[0])
+	require.True(t, isBlob)
+
+	res, err := val.ClientCtx.BroadcastTxSync(btx.Tx)
+	require.NoError(t, err)
```

Comment:
yeah, good point, we should definitely be more consistent. I have been moving towards just using t, mainly because its less of a change between test suite tests and normal tests. also its one less thing we have to declare each test.